src/features/embalm/stepContent/hooks/useCreateSarcophagus/useArchaeologistSignatureNegotiation.ts

```diff
@@ -52,6 +56,7 @@ export function useArchaeologistSignatureNegotiation() {
     async (isRetry: boolean, cancelToken: CancelCreateToken): Promise<void> => {
       console.log('starting the negotiation');
       const lowestRewrapInterval = getLowestRewrapInterval(selectedArchaeologists);
+      const lowestResurrectionTime = getLowestResurrectionTime(selectedArchaeologists);
```

Comment:
Could this be combined with `getLowestRewrapInterval` above to return both `lowestRewrapInterval` and `lowestResurrectionTime` in one go?

Reply:
Yeah, I think this would be a good change.